tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -70,6 +70,8 @@ def exec_cmd(self, **kw):
                     return out[0]
                 return out
 
+            if kw.get("all", False):
```

Comment:
changes needed correspondingly w.r.t to above comment

Reply:
Done